kolibri/core/assets/src/constants.js

```diff
@@ -163,7 +163,7 @@ export const ERROR_CONSTANTS = {
 
 export const DemographicConstants = {
   NOT_SPECIFIED: 'NOT_SPECIFIED',
-  DEFERRED: 'DEFERRED',
+  DEFERRED: '-',
```

Comment:
Hi @estelacruz - thank you for the contribution. We should not be changing the value of the constant here, as this maps to the value returned from the backend.

Instead, we should change how it is displayed only, in the case that the value matches this. We should use the KEmptyPlaceholder component to display the value in that case.

See here for the details of this component: https://design-system.learningequality.org/kemptyplaceholder/

Reply:
@rtibbles I am trying to help @estelacruz but struggling to find where the display components are located in the codebase. Do you by any chance know what file/folder we could find it in?

Reply:
The `KEmptyPlaceholder` component is defined in the `kolibri-design-system` library that is in a [separate repository](https://github.com/learningequality/kolibri-design-system). The components are globally registered, so do not need to be specifically imported.

You can see another example of its use in the codebase here: https://github.com/learningequality/kolibri/blob/f143892a5a70f07fe4424f3d9095029901daf3c5/kolibri/plugins/coach/assets/src/views/common/TruncatedItemList.vue#L3

As you can see, it is not individually imported and registered in the specific component it is used in.